src/server/common/helpers/logging/trace-id-mixin.js

```diff
@@ -0,0 +1,11 @@
+import { getTraceId } from '@defra/hapi-tracing'
+
+export function traceIdMixin() {
```

Comment:
Not convinced on this abstraction. What if you want to add extra things in to the mixin? Considering the mixin pattern is a pino pattern IMO feels like this should just sit where it was in the loggerOptions. So its simple to see and override